### ISSUE TITLE
Zinc throws exception because of corrupted class files

### ISSUE THREAD

**Issue by pfn** [sbt/zinc#156]
```
Caused by: java.lang.RuntimeException: Could not create directory C:\Users\pfnguyen\android-sdk-windows\platforms\android-22\android.jar\android\widget
```

Full stack trace can be found at this gist https://gist.github.com/pfn/8821d2eb5b06a6443ec780048b7f5472

Also, travis logs https://travis-ci.org/scala-android/sbt-android/builds/146283565

Reproduced on https://github.com/scala-android/sbt-android branch `sbt-1.0.0` by running `sbt "scripted gradle-build/materialedittext"`

**Comment by pfn** [comment]
Actually, it's initial build that is failing, the incremental build passes...

```
[pfnguyen@galactica materialedittext] $ sbt clean sample/android:package
[info] Loading project definition from C:\Users\pfnguyen\src\materialedittext\project
Loading cached gradle project definitions
[info] Set current project to materialedittext (in build file:/C:/Users/pfnguyen/src/materialedittext/)
[success] Total time: 0 s, completed Jul 21, 2016 8:28:06 AM
[info] Updating {file:/C:/Users/pfnguyen/src/materialedittext/}library...
[info] Resolving jline#jline;2.12.1 ...
[info] Done updating.
[info] Collecting resources
[info] Performing full resource merge
[info] Processing resources
[info] Rebuilding all classes because R.java has changed
[info] Compiling 11 Java sources to C:\Users\pfnguyen\src\materialedittext\library\target\android\intermediates\classes...
java.lang.IllegalStateException: Failed to parse class class android.widget.TextView$SavedState: this may mean your classfiles are corrupted. Please clean and try again.
        at sbt.internal.inc.ClassToAPI$.fieldToDef(ClassToAPI.scala:180)
        at sbt.internal.inc.ClassToAPI$$anonfun$8.apply(ClassToAPI.scala:95)
        at sbt.internal.inc.ClassToAPI$$anonfun$8.apply(ClassToAPI.scala:95)
        at sbt.internal.inc.ClassToAPI$$anonfun$mergeMap$1.apply(ClassToAPI.scala:245)
        at sbt.internal.inc.ClassToAPI$$anonfun$mergeMap$1.apply(ClassToAPI.scala:245)
        at scala.collection.TraversableLike$$anonfun$flatMap$1.apply(TraversableLike.scala:241)
        at scala.collection.TraversableLike$$anonfun$flatMap$1.apply(TraversableLike.scala:241)
        at scala.collection.IndexedSeqOptimized$class.foreach(IndexedSeqOptimized.scala:33)
        at scala.collection.mutable.WrappedArray.foreach(WrappedArray.scala:35)
        at scala.collection.TraversableLike$class.flatMap(TraversableLike.scala:241)
        at scala.collection.AbstractTraversable.flatMap(Traversable.scala:104)
        at sbt.internal.inc.ClassToAPI$.merge(ClassToAPI.scala:251)
        at sbt.internal.inc.ClassToAPI$.mergeMap(ClassToAPI.scala:245)
        at sbt.internal.inc.ClassToAPI$.structure(ClassToAPI.scala:95)
        at sbt.internal.inc.ClassToAPI$.x$2$lzycompute$1(ClassToAPI.scala:77)
        at sbt.internal.inc.ClassToAPI$.x$2$1(ClassToAPI.scala:77)
        at sbt.internal.inc.ClassToAPI$.instance$lzycompute$1(ClassToAPI.scala:77)
        at sbt.internal.inc.ClassToAPI$.sbt$internal$inc$ClassToAPI$$instance$1(ClassToAPI.scala:77)
        at sbt.internal.inc.ClassToAPI$$anonfun$5.apply(ClassToAPI.scala:78)
        at sbt.internal.inc.ClassToAPI$$anonfun$5.apply(ClassToAPI.scala:78)
        at xsbti.SafeLazy$Impl._t$lzycompute(SafeLazy.scala:18)
        at xsbti.SafeLazy$Impl._t(SafeLazy.scala:16)
        at xsbti.SafeLazy$Impl.get(SafeLazy.scala:22)
        at sbt.internal.inc.ClassToAPI$$anonfun$process$1.apply(ClassToAPI.scala:24)
        at sbt.internal.inc.ClassToAPI$$anonfun$process$1.apply(ClassToAPI.scala:24)
        at scala.collection.immutable.List.foreach(List.scala:381)
        at scala.collection.generic.TraversableForwarder$class.foreach(TraversableForwarder.scala:35)
        at scala.collection.mutable.ListBuffer.foreach(ListBuffer.scala:45)
        at sbt.internal.inc.ClassToAPI$.process(ClassToAPI.scala:24)
        at sbt.internal.inc.javac.AnalyzingJavaCompiler.sbt$internal$inc$javac$AnalyzingJavaCompiler$$readAPI$1(AnalyzingJavaCompiler.scala:74)
        at sbt.internal.inc.javac.AnalyzingJavaCompiler$$anonfun$compile$2$$anonfun$apply$mcV$sp$2$$anonfun$apply$5.apply(AnalyzingJavaCompiler.scala:84)
        at sbt.internal.inc.javac.AnalyzingJavaCompiler$$anonfun$compile$2$$anonfun$apply$mcV$sp$2$$anonfun$apply$5.apply(AnalyzingJavaCompiler.scala:84)
        at sbt.internal.inc.classfile.Analyze$$anonfun$apply$11.apply(Analyze.scala:65)
        at sbt.internal.inc.classfile.Analyze$$anonfun$apply$11.apply(Analyze.scala:60)
        at scala.collection.TraversableLike$WithFilter$$anonfun$foreach$1.apply(TraversableLike.scala:733)
        at scala.collection.mutable.HashMap$$anonfun$foreach$1.apply(HashMap.scala:99)
        at scala.collection.mutable.HashMap$$anonfun$foreach$1.apply(HashMap.scala:99)
        at scala.collection.mutable.HashTable$class.foreachEntry(HashTable.scala:230)
        at scala.collection.mutable.HashMap.foreachEntry(HashMap.scala:40)
        at scala.collection.mutable.HashMap.foreach(HashMap.scala:99)
        at scala.collection.TraversableLike$WithFilter.foreach(TraversableLike.scala:732)
        at sbt.internal.inc.classfile.Analyze$.apply(Analyze.scala:60)
        at sbt.internal.inc.javac.AnalyzingJavaCompiler$$anonfun$compile$2$$anonfun$apply$mcV$sp$2.apply(AnalyzingJavaCompiler.scala:84)
        at sbt.internal.inc.javac.AnalyzingJavaCompiler$$anonfun$compile$2$$anonfun$apply$mcV$sp$2.apply(AnalyzingJavaCompiler.scala:82)
        at scala.collection.TraversableLike$WithFilter$$anonfun$foreach$1.apply(TraversableLike.scala:733)
        at scala.collection.immutable.List.foreach(List.scala:381)
        at scala.collection.TraversableLike$WithFilter.foreach(TraversableLike.scala:732)
        at sbt.internal.inc.javac.AnalyzingJavaCompiler$$anonfun$compile$2.apply$mcV$sp(AnalyzingJavaCompiler.scala:82)
        at sbt.internal.inc.javac.AnalyzingJavaCompiler$$anonfun$compile$2.apply(AnalyzingJavaCompiler.scala:82)
        at sbt.internal.inc.javac.AnalyzingJavaCompiler$$anonfun$compile$2.apply(AnalyzingJavaCompiler.scala:82)
        at sbt.internal.inc.javac.AnalyzingJavaCompiler.timed(AnalyzingJavaCompiler.scala:93)
        at sbt.internal.inc.javac.AnalyzingJavaCompiler.compile(AnalyzingJavaCompiler.scala:81)
        at sbt.internal.inc.MixedAnalyzingCompiler$$anonfun$compileJava$1$1.apply$mcV$sp(MixedAnalyzingCompiler.scala:57)
        at sbt.internal.inc.MixedAnalyzingCompiler$$anonfun$compileJava$1$1.apply(MixedAnalyzingCompiler.scala:57)
        at sbt.internal.inc.MixedAnalyzingCompiler$$anonfun$compileJava$1$1.apply(MixedAnalyzingCompiler.scala:57)
        at sbt.internal.inc.MixedAnalyzingCompiler.timed(MixedAnalyzingCompiler.scala:71)
        at sbt.internal.inc.MixedAnalyzingCompiler.compileJava$1(MixedAnalyzingCompiler.scala:56)
        at sbt.internal.inc.MixedAnalyzingCompiler.compile(MixedAnalyzingCompiler.scala:61)
        at sbt.internal.inc.IncrementalCompilerImpl$$anonfun$compileInternal$1.apply(IncrementalCompilerImpl.scala:162)
        at sbt.internal.inc.IncrementalCompilerImpl$$anonfun$compileInternal$1.apply(IncrementalCompilerImpl.scala:162)
        at sbt.internal.inc.IncrementalCompile$$anonfun$doCompile$1.apply(Compile.scala:78)
        at sbt.internal.inc.IncrementalCompile$$anonfun$doCompile$1.apply(Compile.scala:76)
        at sbt.internal.inc.IncrementalCommon.recompileClasses(IncrementalCommon.scala:63)
        at sbt.internal.inc.IncrementalCommon.cycle(IncrementalCommon.scala:37)
        at sbt.internal.inc.Incremental$$anonfun$1.apply(Incremental.scala:73)
        at sbt.internal.inc.Incremental$$anonfun$1.apply(Incremental.scala:71)
        at sbt.internal.inc.Incremental$.manageClassfiles(Incremental.scala:101)

        at sbt.internal.inc.Incremental$.compile(Incremental.scala:71)
        at sbt.internal.inc.IncrementalCompile$.apply(Compile.scala:59)
        at sbt.internal.inc.IncrementalCompilerImpl.compileInternal(IncrementalCompilerImpl.scala:162)
        at sbt.internal.inc.IncrementalCompilerImpl.incrementalCompile(IncrementalCompilerImpl.scala:107)
        at sbt.internal.inc.IncrementalCompilerImpl.compile(IncrementalCompilerImpl.scala:51)
        at sbt.Compiler$.compile(Compiler.scala:136)
        at sbt.Defaults$.sbt$Defaults$$compileIncrementalTaskImpl(Defaults.scala:891)
        at sbt.Defaults$$anonfun$compileIncrementalTask$1.apply(Defaults.scala:883)
        at sbt.Defaults$$anonfun$compileIncrementalTask$1.apply(Defaults.scala:881)
        at scala.Function1$$anonfun$compose$1.apply(Function1.scala:44)
        at sbt.internal.util.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
        at sbt.std.Transform$$anon$4.work(System.scala:64)
        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:229)
        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:229)
        at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
        at sbt.Execute.work(Execute.scala:238)
        at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:229)
        at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:229)
        at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:161)
        at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.RuntimeException: Could not create directory C:\Users\pfnguyen\android-sdk-windows\platforms\android-22\android.jar\android\widget
        at scala.sys.package$.error(package.scala:27)
        at sbt.io.IO$.createDirectory(IO.scala:190)
        at sbt.io.OpenFile$class.open(Using.scala:44)
        at sbt.io.Using$$anon$2.open(Using.scala:67)
        at sbt.io.Using$$anon$2.open(Using.scala:67)
        at sbt.io.Using.apply(Using.scala:24)
        at sbt.internal.inc.classfile.Parser$.apply(Parser.scala:21)
        at sbt.internal.inc.ClassToAPI$.sbt$internal$inc$ClassToAPI$$classFileForClass(ClassToAPI.scala:111)
        at sbt.internal.inc.ClassToAPI$.cf$lzycompute$1(ClassToAPI.scala:93)
        at sbt.internal.inc.ClassToAPI$.sbt$internal$inc$ClassToAPI$$cf$1(ClassToAPI.scala:93)
        at sbt.internal.inc.ClassToAPI$$anonfun$8$$anonfun$apply$1.apply(ClassToAPI.scala:95)
        at sbt.internal.inc.ClassToAPI$$anonfun$8$$anonfun$apply$1.apply(ClassToAPI.scala:95)
        at sbt.internal.inc.ClassToAPI$.fieldToDef(ClassToAPI.scala:177)
        at sbt.internal.inc.ClassToAPI$$anonfun$8.apply(ClassToAPI.scala:95)
        at sbt.internal.inc.ClassToAPI$$anonfun$8.apply(ClassToAPI.scala:95)
        at sbt.internal.inc.ClassToAPI$$anonfun$mergeMap$1.apply(ClassToAPI.scala:245)
        at sbt.internal.inc.ClassToAPI$$anonfun$mergeMap$1.apply(ClassToAPI.scala:245)
        at scala.collection.TraversableLike$$anonfun$flatMap$1.apply(TraversableLike.scala:241)
        at scala.collection.TraversableLike$$anonfun$flatMap$1.apply(TraversableLike.scala:241)
        at scala.collection.IndexedSeqOptimized$class.foreach(IndexedSeqOptimized.scala:33)
        at scala.collection.mutable.WrappedArray.foreach(WrappedArray.scala:35)
        at scala.collection.TraversableLike$class.flatMap(TraversableLike.scala:241)
        at scala.collection.AbstractTraversable.flatMap(Traversable.scala:104)
        at sbt.internal.inc.ClassToAPI$.merge(ClassToAPI.scala:251)
        at sbt.internal.inc.ClassToAPI$.mergeMap(ClassToAPI.scala:245)
        at sbt.internal.inc.ClassToAPI$.structure(ClassToAPI.scala:95)
        at sbt.internal.inc.ClassToAPI$.x$2$lzycompute$1(ClassToAPI.scala:77)
        at sbt.internal.inc.ClassToAPI$.x$2$1(ClassToAPI.scala:77)
        at sbt.internal.inc.ClassToAPI$.instance$lzycompute$1(ClassToAPI.scala:77)
        at sbt.internal.inc.ClassToAPI$.sbt$internal$inc$ClassToAPI$$instance$1(ClassToAPI.scala:77)
        at sbt.internal.inc.ClassToAPI$$anonfun$5.apply(ClassToAPI.scala:78)
        at sbt.internal.inc.ClassToAPI$$anonfun$5.apply(ClassToAPI.scala:78)
        at xsbti.SafeLazy$Impl._t$lzycompute(SafeLazy.scala:18)
        at xsbti.SafeLazy$Impl._t(SafeLazy.scala:16)
        at xsbti.SafeLazy$Impl.get(SafeLazy.scala:22)
        at sbt.internal.inc.ClassToAPI$$anonfun$process$1.apply(ClassToAPI.scala:24)
        at sbt.internal.inc.ClassToAPI$$anonfun$process$1.apply(ClassToAPI.scala:24)
        at scala.collection.immutable.List.foreach(List.scala:381)
        at scala.collection.generic.TraversableForwarder$class.foreach(TraversableForwarder.scala:35)
        at scala.collection.mutable.ListBuffer.foreach(ListBuffer.scala:45)
        at sbt.internal.inc.ClassToAPI$.process(ClassToAPI.scala:24)
        at sbt.internal.inc.javac.AnalyzingJavaCompiler.sbt$internal$inc$javac$AnalyzingJavaCompiler$$readAPI$1(AnalyzingJavaCompiler.scala:74)
        at sbt.internal.inc.javac.AnalyzingJavaCompiler$$anonfun$compile$2$$anonfun$apply$mcV$sp$2$$anonfun$apply$5.apply(AnalyzingJavaCompiler.scala:84)
        at sbt.internal.inc.javac.AnalyzingJavaCompiler$$anonfun$compile$2$$anonfun$apply$mcV$sp$2$$anonfun$apply$5.apply(AnalyzingJavaCompiler.scala:84)
        at sbt.internal.inc.classfile.Analyze$$anonfun$apply$11.apply(Analyze.scala:65)
        at sbt.internal.inc.classfile.Analyze$$anonfun$apply$11.apply(Analyze.scala:60)
        at scala.collection.TraversableLike$WithFilter$$anonfun$foreach$1.apply(TraversableLike.scala:733)
        at scala.collection.mutable.HashMap$$anonfun$foreach$1.apply(HashMap.scala:99)
        at scala.collection.mutable.HashMap$$anonfun$foreach$1.apply(HashMap.scala:99)
        at scala.collection.mutable.HashTable$class.foreachEntry(HashTable.scala:230)
        at scala.collection.mutable.HashMap.foreachEntry(HashMap.scala:40)
        at scala.collection.mutable.HashMap.foreach(HashMap.scala:99)
        at scala.collection.TraversableLike$WithFilter.foreach(TraversableLike.scala:732)
        at sbt.internal.inc.classfile.Analyze$.apply(Analyze.scala:60)
        at sbt.internal.inc.javac.AnalyzingJavaCompiler$$anonfun$compile$2$$anonfun$apply$mcV$sp$2.apply(AnalyzingJavaCompiler.scala:84)
        at sbt.internal.inc.javac.AnalyzingJavaCompiler$$anonfun$compile$2$$anonfun$apply$mcV$sp$2.apply(AnalyzingJavaCompiler.scala:82)
        at scala.collection.TraversableLike$WithFilter$$anonfun$foreach$1.apply(TraversableLike.scala:733)
        at scala.collection.immutable.List.foreach(List.scala:381)
        at scala.collection.TraversableLike$WithFilter.foreach(TraversableLike.scala:732)
        at sbt.internal.inc.javac.AnalyzingJavaCompiler$$anonfun$compile$2.apply$mcV$sp(AnalyzingJavaCompiler.scala:82)
        at sbt.internal.inc.javac.AnalyzingJavaCompiler$$anonfun$compile$2.apply(AnalyzingJavaCompiler.scala:82)
        at sbt.internal.inc.javac.AnalyzingJavaCompiler$$anonfun$compile$2.apply(AnalyzingJavaCompiler.scala:82)
        at sbt.internal.inc.javac.AnalyzingJavaCompiler.timed(AnalyzingJavaCompiler.scala:93)
        at sbt.internal.inc.javac.AnalyzingJavaCompiler.compile(AnalyzingJavaCompiler.scala:81)
        at sbt.internal.inc.MixedAnalyzingCompiler$$anonfun$compileJava$1$1.apply$mcV$sp(MixedAnalyzingCompiler.scala:57)
        at sbt.internal.inc.MixedAnalyzingCompiler$$anonfun$compileJava$1$1.apply(MixedAnalyzingCompiler.scala:57)
        at sbt.internal.inc.MixedAnalyzingCompiler$$anonfun$compileJava$1$1.apply(MixedAnalyzingCompiler.scala:57)
        at sbt.internal.inc.MixedAnalyzingCompiler.timed(MixedAnalyzingCompiler.scala:71)
        at sbt.internal.inc.MixedAnalyzingCompiler.compileJava$1(MixedAnalyzingCompiler.scala:56)
        at sbt.internal.inc.MixedAnalyzingCompiler.compile(MixedAnalyzingCompiler.scala:61)
        at sbt.internal.inc.IncrementalCompilerImpl$$anonfun$compileInternal$1.apply(IncrementalCompilerImpl.scala:162)
        at sbt.internal.inc.IncrementalCompilerImpl$$anonfun$compileInternal$1.apply(IncrementalCompilerImpl.scala:162)
        at sbt.internal.inc.IncrementalCompile$$anonfun$doCompile$1.apply(Compile.scala:78)
        at sbt.internal.inc.IncrementalCompile$$anonfun$doCompile$1.apply(Compile.scala:76)
        at sbt.internal.inc.IncrementalCommon.recompileClasses(IncrementalCommon.scala:63)
        at sbt.internal.inc.IncrementalCommon.cycle(IncrementalCommon.scala:37)
        at sbt.internal.inc.Incremental$$anonfun$1.apply(Incremental.scala:73)
        at sbt.internal.inc.Incremental$$anonfun$1.apply(Incremental.scala:71)
        at sbt.internal.inc.Incremental$.manageClassfiles(Incremental.scala:101)

        at sbt.internal.inc.Incremental$.compile(Incremental.scala:71)
        at sbt.internal.inc.IncrementalCompile$.apply(Compile.scala:59)
        at sbt.internal.inc.IncrementalCompilerImpl.compileInternal(IncrementalCompilerImpl.scala:162)
        at sbt.internal.inc.IncrementalCompilerImpl.incrementalCompile(IncrementalCompilerImpl.scala:107)
        at sbt.internal.inc.IncrementalCompilerImpl.compile(IncrementalCompilerImpl.scala:51)
        at sbt.Compiler$.compile(Compiler.scala:136)
        at sbt.Defaults$.sbt$Defaults$$compileIncrementalTaskImpl(Defaults.scala:891)
        at sbt.Defaults$$anonfun$compileIncrementalTask$1.apply(Defaults.scala:883)
        at sbt.Defaults$$anonfun$compileIncrementalTask$1.apply(Defaults.scala:881)
        at scala.Function1$$anonfun$compose$1.apply(Function1.scala:44)
        at sbt.internal.util.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
        at sbt.std.Transform$$anon$4.work(System.scala:64)
        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:229)
        at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:229)
        at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
        at sbt.Execute.work(Execute.scala:238)
        at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:229)
        at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:229)
        at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:161)
        at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)
[error] (library/compile:compileIncremental) java.lang.IllegalStateException: Failed to parse class class android.widget.TextView$SavedState: this may mean your classfiles are corrupted. Please clean and try again.
[error] Total time: 4 s, completed Jul 21, 2016 8:28:10 AM
[pfnguyen@galactica materialedittext] $ sbt sample/android:package
[info] Loading project definition from C:\Users\pfnguyen\src\materialedittext\project
Loading cached gradle project definitions
[info] Set current project to materialedittext (in build file:/C:/Users/pfnguyen/src/materialedittext/)
[info] Compiling 11 Java sources to C:\Users\pfnguyen\src\materialedittext\library\target\android\intermediates\classes...
[info] Packaging C:\Users\pfnguyen\src\materialedittext\library\target\android\intermediates\classes.jar ...
[info] Done packaging.
[info] Updating {file:/C:/Users/pfnguyen/src/materialedittext/}sample...
[info] Resolving jline#jline;2.12.1 ...
[info] Done updating.
[info] Collecting resources
[info] Performing full resource merge
[info] Processing resources
[info] Rebuilding all classes because R.java has changed
[info] Compiling 5 Java sources to C:\Users\pfnguyen\src\materialedittext\sample\target\android\intermediates\classes...
[warn] Android SDK updates available, run 'android-update' to update:
[warn]     Intel x86 Atom_64 System Image
[warn]     Intel x86 Atom System Image
[warn]     Intel x86 Atom_64 System Image
[warn]     Intel x86 Atom System Image
[warn]     Intel x86 Atom_64 System Image
[warn]     Google APIs Intel x86 Atom_64 System Image
[warn]     Google APIs Intel x86 Atom System Image
[warn]     Intel x86 Atom System Image
[warn]     Android SDK Platform 24
[warn]     Intel x86 Atom System Image
[info] Packaging resources: resources-debug.ap_
[info] Packaging C:\Users\pfnguyen\src\materialedittext\sample\target\android\intermediates\classes.jar ...
[info] Done packaging.
[info] Generating dex, incremental=true, multidex=false
[info] dex method count: 15791
[info] Packaged: sample-debug-unaligned.apk (1.17MB)
[info] Debug package does not need signing: sample-debug-unaligned.apk
[info] zipaligned: sample-debug.apk
[success] Total time: 9 s, completed Jul 21, 2016 8:28:34 AM
```

**Comment by stuhood** [comment]
@pfn : The fact that it succeeds the second time is due to #144, which we hope to address in the next few weeks.

**Comment by stuhood** [comment]
@pfn : Can you try again post-#158 and see whether you're still seeing this issue?

**Comment by jvican** [comment]
@pfn I tried to reproduce this issue, but I couldn't find the branch `sbt-1.0.0` you talk about in your issue report. Could you either push this branch to the remote repository or try to reproduce the error with a locally published Zinc? As @stuhood points out, #158 may have fixed your issue.

**Comment by jvican** [comment]
There has been no response from @pfn since half a year, so I'm closing under the optimistic assumption that #158 has fixed it. If this is not the case, I encourage the author of this ticket to reopen it.
